services/quests/src/actions/Web.test.tsx

```diff
@@ -0,0 +1,3 @@
+describe('Web action', () => {
+  test('Empty', () => { /* TODO */ });
```

Comment:
Not very applicable in this case since the web action file in QC only contains the handleFetchError helper, which would just be a test against the Fetch API.

What would you think about moving that into a shared /fetch or /requests set of network helpers?

Reply:
SGTM!